typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -320,8 +321,8 @@ export const goerli: ChainMetadata = {
   displayName: 'Goerli',
   nativeToken: etherToken,
   publicRpcUrls: [
-    { http: 'https://eth-goerli.public.blastapi.io' },
     { http: 'https://rpc.ankr.com/eth_goerli' },
+    { http: 'https://eth-goerli.public.blastapi.io' },
```

Comment:
I think this is better, though i think the ankr one can sometimes be stuck too. I wonder if we should just add another one to try https://chainlist.org/chain/5, maybe someone's infura?

Reply:
Okay I'll test the omniatech one: https://endpoints.omniatech.io/v1/eth/goerli/public

Reply:
Works, will use it for now